utils/dataloaders.py

```diff
@@ -704,18 +705,21 @@ def __getitem__(self, index):
         hyp = self.hyp
         mosaic = self.mosaic and random.random() < hyp['mosaic']
         if mosaic:
-            # Load mosaic
-            img, labels = self.load_mosaic(index)
-            shapes = None
-
-            # MixUp augmentation
-            if random.random() < hyp['mixup']:
-                img, labels = mixup(img, labels, *self.load_mosaic(random.randint(0, self.n - 1)))
-
+            # # Load mosaic
+            # img, labels = self.load_mosaic(index)  # TODO im_orig
+            # shapes = None
+            #
+            # # MixUp augmentation
+            # if random.random() < hyp['mixup']:
+            #     img, labels = mixup(img, labels, *self.load_mosaic(random.randint(0, self.n - 1)))
+
+            # Raise an exception to indicate that the 'mosaic' hyperparameter is not supported
+            raise NotImplementedError(
```

Comment:
let's uncomment these lines and move the `raise` right after `if mosaic` --> less changes

Reply:
Oke!